examples/inventory_example.rb

```diff
@@ -26,7 +26,7 @@ def print_object(base, caption, indent = 0, recurse = true)
         puts "#{indentation}Ignoring #{rel} relationship due to ovirt error: #{err}"
       rescue RestClient::InternalServerError => err
         puts_caption(new_caption, indentation)
-        puts "#{indentation}Ignoring #{rel} relationship due to rest client error: #{err}"
+        puts "#{indentation}Ignoring #{rel} relationship due to rest client error: #{err.response}"
```

Comment:
@bdunne  Thanks, I understand now.   I know this is `print_object` so I assume this is for development/debugging but is there a possibility the response will have anything in it we need to mask/hide?